### PR TITLE
Contract Module Integration Tests

### LINKED ISSUE #440: Add integration tests to the contract module
The contract module currently only has unit tests. Integration tests should be added for these classes.

### PR DESCRIPTION
**Pull request by mialbu**
Resolves #440.

- [x] ContractManagement
- [x] FungibleToken
- [x] NeoToken / GasToken
- [x] NeoNameService / NonFungibleToken
- [x] NeoURI
- [x] PolicyContract
- [x] RoleManagement
- [x] Token
- SmartContract: automatically covered by other tests
- TransactionBuilder: automatically covered by other tests - may be discussed if further integration testing is needed.

### CODE REVIEW

**Comment by gsmachado** [comment]
Great stuff, @mialbu! 💪 👍 💯 

We really need these tests, indeed.

File: contract/src/test-integration/java/io/neow3j/contract/NeoTokenIntegrationTest.java [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+                .getSendRawTransaction()
+                .getHash();
+        waitUntilTransactionIsExecuted(txHash, neow3j);
+
+        candidates = neoToken.getCandidates();
+        assertThat(candidates.entrySet(), hasSize(0));
+    }
+
+    @Test
+    public void testRegisterAndVote() throws Throwable {
```
Comment (csmuller) [inline contract/src/test-integration/java/io/neow3j/contract/NeoTokenIntegrationTest.java:86]: This test has no asserts.

File: contract/src/test-integration/java/io/neow3j/contract/ContractManagementIntegrationTest.java [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    private static Neow3j neow3j;
+    private static ContractManagement contractManagement;
+
+    private final static Path TESTCONTRACT_NEF_FILE = Paths.get("/contracts", "TestContract.nef");
+    private final static Path TESTCONTRACT_MANIFEST_FILE =
+            Paths.get("/contracts", "TestContract.manifest.json");
+
+    @ClassRule
+    public static NeoTestContainer neoTestContainer =
+            new NeoTestContainer("/node-config/config.json");
```
Comment (csmuller) [inline contract/src/test-integration/java/io/neow3j/contract/ContractManagementIntegrationTest.java:39]: In all the test classes it is no necessary to specify the path to the config.json if it has the default name, as it has here. The default constructor already uses this path.

**Review by csmuller — CHANGES_REQUESTED** [review]

File: contract/src/test-integration/java/io/neow3j/contract/NeoTokenIntegrationTest.java (hunk 1 above)
Comment (mialbu) [reply contract/src/test-integration/java/io/neow3j/contract/NeoTokenIntegrationTest.java:91]: I fixed this test in the commit b39a19d. Have a look. :relaxed:

**Review by csmuller — APPROVED** [review]

**Comment by mialbu** [comment]
This was done in pr #495.

**Pull request merged**